### PR TITLE
Donate Button V4

### PR DESCRIPTION
**Pull request by Fishbakh-N**
**Screenshot** 
image

**New widget options:**

```ts
{
        nonprofitSlug: string;
	fundraiserSlug?: string;
	methods: PaymentMethod[];
	openAt: string;
	show: boolean;
	primaryColor: string;
	defaultDonationAmount?: number;
	minDonationAmount: number;
	defaultFrequency: DonationFrequency;
	addAmounts?: number[];
	completeDonationInNewTab?: boolean;
	noExit?: boolean;
}

PaymentMethod {
	'card',
	'bank',
	'paypal',
	'venmo',
	'pay', // google/apple pay
	'crypto',
	'stocks',
	'daf'
}

DonationFrequency {
	'monthly',
	'once',
}
```

### CODE REVIEW

**Comment by markulrich** [comment]
Exciting! @Fishbakh-N can you make it so that at the demo index https://donate-button-jny9z0pi9-everydotorg.vercel.app/ 
image


There is a link to 0.4?

**Comment by markulrich** [comment]
Also we need to make sure that the 0.3 code does not get deleted at all ... maybe we should put all this new code in a new folder `packages/donate-button-04/` ?? Seems like the easiest solution to ensure the many websites using the 0.3 library do not have any trouble

File: packages/donate-button-next/src/components/widget/components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {footerCardCss} from 'src/components/widget/components/Footer/styles';
+import {GridCard} from 'src/components/widget/components/GridCard';
+
+export const Footer = () => {
+	return (
+		<GridCard className={footerCardCss}>
+			<p>
+				100% of your donation is tax-deductible to the extent allowed by US law.
+				Your donation is made to Every.org, a tax-exempt US 501(c)(3) charity
+				that grants unrestricted funds to SDAF on your behalf. As a legal
```
Comment (markulrich) [inline packages/donate-button-next/src/components/widget/components/Footer/index.tsx:10]: To {nonprofit.name} not to SDAF.

Ideally, should replicate the logic we have for `getTaxDeductibleStatement` in the main repo

**Comment by markulrich** [comment]
Nice!!!!

In the upper left, instead of:

image


Please use:

image

(For the color, fine to just be a white background.)

**Comment by markulrich** [comment]
Looking good!!! Two questions
1) Could we add the beginning of Apple/Google Pay? Could you auto-detect support just using Javascript? (Not the Stripe libraries)
2) Why is there a "loading" pause for 0.4, vs the 0.3 was instant? I think the animation looks good, but could we make it shorter?

**Comment by Fishbakh-N** [comment]
@mark 
1. ОК, I'll look into it. I'll see what I can do about it.
2. in 0.3 you can provide name, logo url, description, address by passing it as props for widget. in 0.4 it only `primarySlug` and all data will be loaded after

**Review by markulrich — APPROVED** [review]
Fantastic!! Trying it out things were great :)

I did not have a chance to read the code carefully, @martinbianchi and @OlegKrav64 could you review in more detail?

**Comment by OlegKrav64** [comment]
Looks good !

**Pull request merged**